server/tests/integration/graphql.test.js

```diff
@@ -181,444 +181,444 @@ describe('graphql', () => {
     // eslint-disable-next-line no-unused-vars
     const queryResult = await typeAwareQuery(
       gql`
-                query {
+        query {
+          __typename
+          browsers {
+            __typename
+            id
+            key
+            name
+            ats {
+              __typename
+              id
+              name
+            }
+            candidateAts {
+              __typename
+              id
+              name
+            }
+            recommendedAts {
+              __typename
+              id
+              name
+            }
+            browserVersions {
+              __typename
+              id
+              name
+            }
+          }
+          ats {
+            __typename
+            id
+            key
+            name
+            browsers {
+              __typename
+              id
+              name
+            }
+            candidateBrowsers {
+              __typename
+              id
+              name
+            }
+            recommendedBrowsers {
+              __typename
+              id
+              name
+            }
+            atVersions {
+              __typename
+              id
+              name
+              releasedAt
+            }
+          }
+          users {
+            __typename
+            username
+            roles
+            isBot
+          }
+          me {
+            __typename
+            id
+            username
+            roles
+            ats {
+              id
+              name
+            }
+          }
+          collectionJob(id: 1) {
+            __typename
+            id
+            status
+            testPlanRun {
+              id
+            }
+          }
+          collectionJobs {
+            __typename
+            id
+            status
+          }
+          collectionJobByTestPlanRunId(testPlanRunId: 1) {
+            __typename
+            id
+            status
+          }
+          v2TestPlanVersion: testPlanVersion(id: 80) {
```

Comment:
For reviewers: Formatting has messed with this diff. This id was changed to 80 from 133